From 791e8139c52462b32c62733f5e2a46d393f7d441 Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh@glandium.org>
Date: Thu, 1 Apr 2010 15:34:59 +0200
Subject: [PATCH 13/15] Use googpub-phish-shavar instead of goog-phish-shavar for safe browsing

This is a first step for fixing safe browsing. Either the current setup will work
if Google allows our client string to get this data, or we'll be allowed to say
we're firefox to the safe browsing server.
---
 browser/app/profile/firefox.js                     |    2 +-
 .../components/safebrowsing/content/application.js |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index e3f80a9..2803671 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -697,7 +697,7 @@ pref("urlclassifier.alternate_error_page", "blocked");
 pref("urlclassifier.gethashnoise", 4);
 
 // The list of tables that use the gethash request to confirm partial results.
-pref("urlclassifier.gethashtables", "goog-phish-shavar,goog-malware-shavar");
+pref("urlclassifier.gethashtables", "googpub-phish-shavar,goog-malware-shavar");
 
 // If an urlclassifier table has not been updated in this number of seconds,
 // a gethash request will be forced to check that the result is still in
diff --git a/browser/components/safebrowsing/content/application.js b/browser/components/safebrowsing/content/application.js
index f5be69f..66821f8 100644
--- a/browser/components/safebrowsing/content/application.js
+++ b/browser/components/safebrowsing/content/application.js
@@ -109,7 +109,7 @@ PROT_Application.prototype.initialize = function() {
   this.malwareWarden.maybeToggleUpdateChecking();
 
   this.phishWarden = new PROT_PhishingWarden();
-  this.phishWarden.registerBlackTable("goog-phish-shavar");
+  this.phishWarden.registerBlackTable("googpub-phish-shavar");
   this.phishWarden.maybeToggleUpdateChecking();
 }
 
-- 
1.7.0.rc0.69.g2ee80

